app/serializers/api/v1/book_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  module V1
+    class BookSerializer < ActiveModel::Serializer
```

Comment:
Are you only returning id?

Reply:
Yes! I implemented it in the book-test-rent-index-show-paginate-test-book-index branch.

Reply:
With attributes: :id, :genre, :author, :image, :title, :publisher, :year, :created_at, :updated_at